### PR TITLE
xz: add page

### PR DESCRIPTION
**Pull request by oxguy3**
[xz](http://tukaani.org/xz/) is a compression tool.

### CODE REVIEW

File: pages/common/xz.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Decompress a file and write to stdout:
+
+`xz -dc {{file.xz}}`
+
+- Compress a file, but don't delete the original:
+
+`xz -k {{file}}`
+
+- Compress a file and attempt to improve compression ratio by using more CPU time:
```
Comment (rubenvereecken) [inline pages/common/xz.md]: You have a lot of different ways of saying "better but slower" among your many, many compression pages. Might want to stick to one ;).

Comment (oxguy3) [inline pages/common/xz.md]: I tend to sort of copy the wording the original man page uses. :stuck_out_tongue: Actually, I'm realizing that the `-e` flag doesn't make sense without a `-0`...`-9` flag as well. I'm gonna drop this example and use those instead (I'm typically too lazy to use anything but default compression ratio, but I imagine anyone interested enough in compression to not just use the more-popular gzip might care about compression ratios).

File: pages/common/xz.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Decompress a file and write to stdout:
+
+`xz -dc {{file.xz}}`
+
+- Compress a file, but don't delete the original:
+
+`xz -k {{file}}`
+
+- Compress a file using fastest compression:
```
Comment (igorshubovych) [inline pages/common/xz.md]: *the* fastest

File: pages/common/xz.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`xz -0 {{file}}`
+
+- Compress a file using best compression:
```
Comment (igorshubovych) [inline pages/common/xz.md]: *the* best

**Comment by igorshubovych** [comment]
LGTM. Thanks!

**Pull request merged**